### PR TITLE
[docs][s]: added tutorials sections with tutorial on how to build gra…

### PR DESCRIPTION
**Pull request by Mikanebu**
* added `tutorials` section
  * added how to build graphs and visualizations page

### CODE REVIEW

**Comment by Mikanebu** [comment]
screen shot 2017-12-28 at 2 28 25 am

File: views/docs_home.html [hunk 1]
```diff
@@ -48,6 +48,66 @@ <h4>Getting data tutorial</h4>
...
+    <!-- Tutorials -->
+    <div class="docs-section">
+      <h2>
+        Tutorials
+      </h2>
+      <p class="description">
+        Learn DataHub features by tutorials!
+      </p>
+      <div class="row">
+        <a href="http://datahub.io/examples/simple-graph-spec" class="section-link">
```
Comment (anuveyatsu) [inline views/docs_home.html]: can we use `https` instead of `http`

File: views/docs_home.html [hunk 2]
```diff
@@ -48,6 +48,66 @@ <h4>Getting data tutorial</h4>
...
+        <a href="http://datahub.io/examples/vega-views-tutorial-grouping" class="section-link">
+          <div class="col-md-4">
+            <h4>Grouping</h4>
+            <p>Tutorial on how to build visualizations using the <b>Vega Graph Spec</b></p>
+          </div>
+        </a>
+        <a href="http://datahub.io/examples/transform-example-gdp-uk" class="section-link">
+          <div class="col-md-4">
+            <h4>Transform</h4>
+            <p>Tutorial on how data transforms work</p>
```
Comment (anuveyatsu) [inline views/docs_home.html]: This is not useful and you also have repeated it 3 times

File: views/docs_home.html [hunk 3]
```diff
@@ -48,6 +48,66 @@ <h4>Getting data tutorial</h4>
...
+      <div class="row">
+        <a href="https://datahub.io/examples/simple-graph-spec" class="section-link">
+          <div class="col-md-4">
+            <h4>Graphs</h4>
+            <p>Tutorial on how to build the simple and nice graphs using the <b>Simple Graph Spec</b>. </p>
+          </div>
+        </a>
+        <a href="https://datahub.io/examples/geojson-tutorial" class="section-link">
+          <div class="col-md-4">
+            <h4>GeoJSON</h4>
```
Comment (anuveyatsu) [inline views/docs_home.html]: remove GeoJSON for now as it does not work

File: views/docs_home.html (hunk 1 above)
Comment (Mikanebu) [reply views/docs_home.html]: done

File: views/docs_home.html (hunk 2 above)
Comment (Mikanebu) [reply views/docs_home.html]: Do you have any suggestion? It would be nice to keep only one of them

Comment (anuveyatsu) [reply views/docs_home.html]: @Mikanebu they all are different, and next time you should not just have identical desctiprions

Comment (anuveyatsu) [reply views/docs_home.html]: "In this example, we explain how to aggregate data."

File: views/docs_home.html [hunk 4]
```diff
@@ -48,6 +48,66 @@ <h4>Getting data tutorial</h4>
...
+        </a>
+        <a href="http://datahub.io/examples/vega-views-tutorial-grouping" class="section-link">
+          <div class="col-md-4">
+            <h4>Grouping</h4>
+            <p>Tutorial on how to build visualizations using the <b>Vega Graph Spec</b></p>
+          </div>
+        </a>
+        <a href="http://datahub.io/examples/transform-example-gdp-uk" class="section-link">
+          <div class="col-md-4">
+            <h4>Transform</h4>
```
Comment (anuveyatsu) [inline views/docs_home.html]: Not just "Transform"... but may be "Data Transform - Aggregation"

File: views/docs_home.html [hunk 5]
```diff
@@ -48,6 +48,66 @@ <h4>Getting data tutorial</h4>
...
+            <p>Tutorial on how data transforms work</p>
+          </div>
+        </a>
+        <a href="http://datahub.io/examples/example-sample-transform-on-currency-codes" class="section-link">
+          <div class="col-md-8">
+            <h4>Transform</h4>
```
Comment (anuveyatsu) [inline views/docs_home.html]: "Data Transform - Sampling"

File: views/docs_home.html [hunk 6]
```diff
@@ -48,6 +48,66 @@ <h4>Getting data tutorial</h4>
...
+            <p>Tutorial on how data transforms works</p>
```
Comment (anuveyatsu) [inline views/docs_home.html]: Learn how to get a sample of your data.

File: views/docs_home.html [hunk 7]
```diff
@@ -48,6 +48,66 @@ <h4>Getting data tutorial</h4>
...
+          </div>
+        </a>
+        <a href="http://datahub.io/examples/transform-examples-on-co2-fossil-global" class="section-link">
+          <div class="col-md-4">
+            <h4>Transform</h4>
```
Comment (anuveyatsu) [inline views/docs_home.html]: Data Transform - Filtering and Applying Formula

File: views/docs_home.html [hunk 8]
```diff
@@ -48,6 +48,66 @@ <h4>Getting data tutorial</h4>
...
+            <p>Tutorial on how data transforms works</p>
```
Comment (anuveyatsu) [inline views/docs_home.html]: In this example, we explain how filtering and applying formula can be done before dataset gets rendered in showcase page.

File: views/docs_home.html [hunk 9]
```diff
@@ -48,6 +48,66 @@ <h4>Getting data tutorial</h4>
...
+        <a href="http://datahub.io/examples/vega-views-tutorial-topojson" class="section-link">
+          <div class="col-md-4">
+            <h4>TopoJSON</h4>
+            <p>Tutorial on how to build visualizations using the <b>Vega Graph Spec</b></p>
+          </div>
+        </a>
+        <a href="http://datahub.io/examples/vega-views-tutorial-grouping" class="section-link">
+          <div class="col-md-4">
+            <h4>Grouping</h4>
+            <p>Tutorial on how to build visualizations using the <b>Vega Graph Spec</b></p>
```
Comment (anuveyatsu) [inline views/docs_home.html]: this does not have any connection with its title

File: views/docs_home.html [hunk 10]
```diff
@@ -48,6 +48,66 @@ <h4>Getting data tutorial</h4>
...
+        </a>
+        <a href="http://datahub.io/examples/vega-views-tutorial-lines" class="section-link">
+          <div class="col-md-4">
+            <h4>Visualizations</h4>
+            <p>Tutorial on how to build visualizations using the <b>Vega Graph Spec</b></p>
+          </div>
+        </a>
+        <a href="http://datahub.io/examples/vega-views-tutorial-topojson" class="section-link">
+          <div class="col-md-4">
+            <h4>TopoJSON</h4>
```
Comment (anuveyatsu) [inline views/docs_home.html]: Not sure if title should be topojson

File: views/docs_home.html [hunk 11]
```diff
@@ -48,6 +48,66 @@ <h4>Getting data tutorial</h4>
...
+        </a>
+        <a href="http://datahub.io/examples/geojson-tutorial" class="section-link">
+          <div class="col-md-4">
+            <h4>GeoJSON</h4>
+            <p>Tutorial on how to package up <b>GeoJSON</b> data and display it on the map.. </p>
+          </div>
+        </a>
+        <a href="http://datahub.io/examples/vega-views-tutorial-lines" class="section-link">
+          <div class="col-md-4">
+            <h4>Visualizations</h4>
```
Comment (anuveyatsu) [inline views/docs_home.html]: why this one is visualizations but others not?

File: views/docs_home.html [hunk 12]
```diff
@@ -48,6 +48,66 @@ <h4>Getting data tutorial</h4>
...
+            <p>Tutorial on how to build visualizations using the <b>Vega Graph Spec</b></p>
```
Comment (anuveyatsu) [inline views/docs_home.html]: Again you repeated same description 3 times

File: views/docs_home.html [hunk 13]
```diff
@@ -48,6 +48,66 @@ <h4>Getting data tutorial</h4>
...
+      <h2>
+        Tutorials
+      </h2>
+      <p class="description">
+        Learn DataHub features by tutorials!
+      </p>
+      <div class="row">
+        <a href="http://datahub.io/examples/simple-graph-spec" class="section-link">
+          <div class="col-md-4">
+            <h4>Graphs</h4>
```
Comment (anuveyatsu) [inline views/docs_home.html]: This title is very confusing

**Comment by Mikanebu** [comment]
screen shot 2017-12-28 at 9 32 12 pm

**Pull request merged**